SIL.Core/Progress/XslTransformWithProgress.cs

```diff
@@ -109,18 +109,6 @@ public void Transform(ProgressState progressState)
 					_progressState.WriteToLog(err.Message);
 					_progressState.State = ProgressState.StateValue.StoppedWithError;
 				}
-				finally
```

Comment:
So transforms clean up their own temp files now?

Reply:
I'm not sure. This TemporaryFiles property was removed in .NET Core and .NET Standard, so there is no way to cleanup the files.

Reply:
I added the code back with conditional compilation.